src/app/app-config/app-config.model.ts

```diff
@@ -26,4 +26,10 @@ export interface IAppConfig {
     outdated_threshold_days: number;
     useTemporaryDatabase: boolean;
   };
+
+  webdav: {
+    remote_url: string;
+    username: string;
```

Comment:
This poses a security risk - even without logging in the webdav credentials can be seen, so files readable & writable by anyone. Not sure how we can resolve this - it must be done dynamically only after user authorisation somehow ...

Reply:
Agreed. Were you able to get the nextcloud running?

Reply:
We could put this into the couchdb as a config record instead. Then at least an initial successful login is required to access it.

Also, I've considered creating a second database besides the one containing all the entities that would hold all configuration and would be readonly for normal users, requiring special permissions to change anything.

Reply:
Now an encrypted version of the password gets saved, which can be accessed by decrypting with the user password.

Reply:
Currently the cloud-service password has to be entered manually on the user-account page. It then gets encrypted with the login password as key and afterwards the encrypted password gets saved in the database.